File: iOS/PanoAudioChat/PanoAudioChatSDK/AppConfig.m
//
//  AppConfig.m
//  PanoAudioChatSDK
//
//  Created by pano on 2021/7/21.
//

#import "AppConfig.h"

/**
 1. 请输入PanoAppID，PanoServerURL
 2. 参考 https://console.pano.video/#/user/login
 */

static NSString *PanoAppID = <#T##PanoAppID: String##String#>;

static NSString *PanoRtcServer = @"api.pano.video";

@implementation AppConfig

+ (NSString *)appID {
    return PanoAppID;
}

+ (NSString *)rtcServerURL {
    return PanoRtcServer;
}

@end

File: iOS/PanoAudioChat/PanoAudioChat/Classes/OpenApi/AppConfig.swift
//
//  AppConfig.swift
//  PanoAudioChat
//
//  Created by wenhua.zhang on 2021/6/11.
//

import Foundation

/**
 1. 请输入AppID，PanoServerURL
 2. 参考 https://console.pano.video/#/user/login
 */
@objc final class AppConfig: NSObject {
    
    static let PanoAppID: String = <#T##PanoAppID: String##String#>;
    
    static var PanoRtcServerURL: String = "api.pano.video";
}
